.github/workflows/_win-test.yml

```diff
@@ -161,8 +161,8 @@ jobs:
           PYTORCH_TEST_RERUN_DISABLED_TESTS: ${{ matrix.rerun_disabled_tests && '1' || '0' }}
         run: |
           pushd "${PYTORCH_FINAL_PACKAGE_DIR}"
-          # shellcheck disable=SC2046
-          python3 -mpip install $(echo *.whl)[opt-einsum]
+          # shellcheck disable=SC2046,SC2102
+          python3 -mpip install $(echo *.whl)[opt-einsum,optree]
```

Comment:
Why this is necessary, as we don't do any `torch.compile` testing on Windows?

Reply:
The final goal for this migration is to replace all pure-Python-based PyTree implementation (`torch.utils._pytree`), which is already used in some downstream packages (e.g., `torchvision`). I will add tests for `optree` in some follow-up commits. See also my comment https://github.com/pytorch/pytorch/pull/93139#issuecomment-1722141585.